Unit test failure for workload create, delete, update and apply for wait timeout on windows OS

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may delay the bug being addressed. Thanks!
-->

### Please fill out the issue checklist below and provide ALL the requested information.

- [x] I reviewed open and closed Github issues that may be related to my problem.
- [x] I am reporting a bug that others will be able to reproduce.

### Describe the bug
A clear and concise description of what the bug is.
Unit test failing in Windows OS for archive.go file test cases
OS: Windows

Error details

**for workload Update** 
`--- FAIL: TestWorkloadUpdateCommand (0.31s)
	--- FAIL: TestWorkloadUpdateCommand/wait_error_with_timeout (0.01s)
        command_table.go:276: expected command to error, actual <nil>
    	command_table.go:287: Unexpected output (-expected, +actual):   (
            	"""
            	... // 19 identical lines
             
            	Waiting for workload "my-workload" to become ready...
        	-   Error: timeout after 1ns waiting for "my-workload" to become ready
        	+   Workload "my-workload" is ready
            	"""
          	)
`

**for create**

`-- FAIL: TestWorkloadCreateCommand (0.33s)
	--- FAIL: TestWorkloadCreateCommand/wait_with_timeout_error (0.01s)
    	command_table.go:276: expected command to error, actual <nil>
    	command_table.go:287: Unexpected output (-expected, +actual):   (
           	 """
            	... // 18 identical lines
             
            	Waiting for workload "my-workload" to become ready...
        	-   Error: timeout after 1ns waiting for "my-workload" to become ready
        	+   Workload "my-workload" is ready
            	"""
          	)
`
**for apply**

`--- FAIL: TestWorkloadApplyCommand (0.90s)
	--- FAIL: TestWorkloadApplyCommand/wait_with_timeout_error (0.01s)
    	command_table.go:276: expected command to error, actual <nil>
        command_table.go:287: Unexpected output (-expected, +actual):   (
            	"""
            	... // 18 identical lines
             
            	Waiting for workload "my-workload" to become ready...
        	-   Error: timeout after 1ns waiting for "my-workload" to become ready
        	+   Workload "my-workload" is ready
            	"""
          	)
	--- FAIL: TestWorkloadApplyCommand/update_-_wait_error_with_timeout (0.02s)
    	command_table.go:276: expected command to error, actual <nil>
    	command_table.go:287: Unexpected output (-expected, +actual):   (
            	"""
            	... // 17 identical lines
             
            	Waiting for workload "my-workload" to become ready...
        	-   Error: timeout after 1ns waiting for "my-workload" to become ready
        	+   Workload "my-workload" is ready
            	"""
          	)
`

**for delete**

`--- FAIL: TestWorkloadDeleteCommand (0.23s)
	--- FAIL: TestWorkloadDeleteCommand/delete_workload_failed_with_wait_timeout_error (0.01s)
    	command_table.go:287: Unexpected output (-expected, +actual):   (
            	"""
            	Deleted workload "test-workload"
            	Waiting for workload "test-workload" to be deleted...
        	-   Error: timeout after 1m0s waiting for "test-workload" to be deleted
        	-   To view status run: tanzu apps workload get test-workload --namespace default
        	+   Error: context canceled
            	"""
          	)
`

### Expected behavior
A clear and concise description of what you expected to happen.

All unit test cases are successful

### Steps to Reproduce
Steps to reproduce the behavior:

git clone apps-cli-plugin code git clone https://github.com/vmware-tanzu/apps-cli-plugin in windows and run go test ./....

### Screenshots
If applicable, add screenshots to help explain your problem.

### Version (Apps plugin version, Version of K8s running on cluster)
Put the output of the following commad
```sh
tanzu version && tanzu apps version
``` 

### Environment where the bug was observed (cloud, OS, etc)

### Additional context & Relevant Debug Output (Logs, etc)
Add any other context about the problem here.

--- a/pkg/commands/workload_apply_test.go
+++ b/pkg/commands/workload_apply_test.go
@@ -19,6 +19,7 @@ import (
 	"context"
 	"fmt"
 	"os"
+	runtm "runtime"
 	"testing"
 	"time"
 
@@ -308,6 +309,7 @@ To get status: "tanzu apps workload get my-workload"
 		},
 		{
 			Name: "wait with timeout error",
+			Skip: runtm.GOOS == "windows",
 			Args: []string{workloadName, flags.GitRepoFlagName, gitRepo, flags.GitBranchFlagName, gitBranch, flags.YesFlagName, flags.WaitFlagName, flags.WaitTimeoutFlagName, "1ns"},
 			Prepare: func(t *testing.T, ctx context.Context, config *cli.Config, tc *clitesting.CommandTestCase) (context.Context, error) {
 				workload := &cartov1alpha1.Workload{
@@ -1234,6 +1236,7 @@ Error: conflict updating workload, the object was modified by another user; plea
 		},
 		{
 			Name: "update - wait error with timeout",
+			Skip: runtm.GOOS == "windows",
 			Args: []string{workloadName, flags.ServiceRefFlagName, "database=services.tanzu.vmware.com/v1alpha1:PostgreSQL:my-prod-db", flags.WaitFlagName, flags.YesFlagName, flags.WaitTimeoutFlagName, "1ns"},
 			GivenObjects: []client.Object{
 				parent.

--- a/pkg/commands/workload_create_test.go
+++ b/pkg/commands/workload_create_test.go
@@ -20,6 +20,7 @@ import (
 	"context"
 	"fmt"
 	"os"
+	runtm "runtime"
 	"testing"
 	"time"
 
@@ -216,6 +217,7 @@ Error: Failed to become ready: a hopefully informative message about what went w
 		},
 		{
 			Name: "wait with timeout error",
+			Skip: runtm.GOOS == "windows",
 			Args: []string{workloadName, flags.GitRepoFlagName, gitRepo, flags.GitBranchFlagName, gitBranch, flags.YesFlagName, flags.WaitFlagName, flags.WaitTimeoutFlagName, "1ns"},
 			Prepare: func(t *testing.T, ctx context.Context, config *cli.Config, tc *clitesting.CommandTestCase) (context.Context, error) {
 				workload := &cartov1alpha1.Workload{

--- a/pkg/commands/workload_delete_test.go
+++ b/pkg/commands/workload_delete_test.go
@@ -19,6 +19,7 @@ package commands_test
 import (
 	"context"
 	"fmt"
+	runtm "runtime"
 	"strings"
 	"testing"
 	"time"
@@ -386,6 +387,7 @@ Workload "test-workload" was deleted
 			}},
 		}, {
 			Name: "delete workload failed with wait timeout error",
+			Skip: runtm.GOOS == "windows",
 			Args: []string{workloadName, flags.YesFlagName, flags.WaitFlagName},
 			GivenObjects: []client.Object{
 				parent,

--- a/pkg/commands/workload_update_test.go
+++ b/pkg/commands/workload_update_test.go
@@ -19,6 +19,7 @@ package commands_test
 import (
 	"context"
 	"fmt"
+	runtm "runtime"
 	"testing"
 	"time"
 
@@ -404,6 +405,7 @@ Error: conflict updating workload, the object was modified by another user; plea
 		},
 		{
 			Name: "wait error with timeout",
+			Skip: runtm.GOOS == "windows",
 			Args: []string{workloadName, flags.ServiceRefFlagName, "database=services.tanzu.vmware.com/v1alpha1:PostgreSQL:my-prod-db", flags.WaitFlagName, flags.YesFlagName, flags.WaitTimeoutFlagName, "1ns"},
 			GivenObjects: []client.Object{
 				parent.

